nemo/collections/asr/parts/utils/decoder_timestamps_utils.py

```diff
@@ -23,8 +23,8 @@
 from omegaconf import OmegaConf
 
 import nemo.collections.asr as nemo_asr
-from nemo.collections.asr.metrics.wer import WER
-from nemo.collections.asr.metrics.wer_bpe import WERBPE
+from nemo.collections.asr.metrics.wer import WER, CTCCharDecoding, CTCCharDecodingConfig
+from nemo.collections.asr.metrics.wer_bpe import WERBPE, CTCBPEDecoding, CTCBPEDecodingConfig
```

Comment:
Do we have somewhere in the tutorials or docs on how to find the time stamps of the chars or words with ASR models? Something like the usage section of this PR but for both CTC and RNNT models.
If yes, maybe we can add this feature to the readme? as I see many people are looking for this feature and they may not be aware it exists or how to do it.

Reply:
We have timestamp extraction tutorial as a part of "ASR+diarization" tutorial. 
https://github.com/NVIDIA/NeMo/blob/main/tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb
But this may not be the most friendly way to demonstrate it since it is diarization focused. If necessary, we can have timestamp extraction as ASR.  

In addition, only the start of the word is usually accurate and the end of the word is hard to be estimated.

Reply:
I can update the CTC chad and subword tutorials to add sections for the decoding strategy. Let's do that in separate pr.